src/ha-health-check/ha_health_check.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const http = require('http');
+const {
+  spawnSync
+} = require('child_process');
+
+http.createServer(function (req, res) {
+  const ls = spawnSync('bash', ['/var/vcap/jobs/keepalived/bin/get_keepalived_state.sh']);
+  if (ls.stdout.toString().trim() === 'up') {
+    res.writeHead(200, { 'Content-Type': 'text/html' });
```

Comment:
Shouldn't the response content type be `text/plain` rather than `text/html`

Reply:
done!